pkgs/applications/office/ledger-autosync/default.nix

```diff
@@ -2,16 +2,24 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "ledger-autosync";
-  version = "1.0.2";
+  version = "1.0.2-0b674c";
```

Comment:
I think generally we use `unstable-YYYY-MM-DD` for such situations when we don't have a pre-release fetched from git (like `nixUnstable`) or a release-candidate. I could live with those however if they are an established convention in nixpkgs already (or pythonPackages?). Just never saw this format before.

Reply:
That's what is recommended by our manual in https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming

Reply:
You are right, fixed.